README.md

```diff
@@ -0,0 +1,207 @@
+
+In [12]: print(my_plan)
+relations {
+  root {
+    input {
+      project {
+        common {
+          emit {
+            output_mapping: 3
+            output_mapping: 4
+            output_mapping: 5
+          }
+        }
+        input {
+          read {
+            common {
+              direct {
+              }
+            }
+            base_schema {
+              names: "tconst"
+              names: "averageRating"
+              names: "numVotes"
+              struct {
+                types {
+                  string {
+                    nullability: NULLABILITY_NULLABLE
+                  }
+                }
+                types {
+                  string {
+                    nullability: NULLABILITY_NULLABLE
+                  }
+                }
+                types {
+                  string {
+                    nullability: NULLABILITY_NULLABLE
+                  }
+                }
+                nullability: NULLABILITY_REQUIRED
+              }
+            }
+            named_table {
+              names: "ratings"
+            }
+          }
+        }
+        expressions {
+          selection {
+            direct_reference {
+              struct_field {
+              }
+            }
+            root_reference {
+            }
+          }
+        }
+        expressions {
+          cast {
+            type {
+              fp64 {
+                nullability: NULLABILITY_NULLABLE
+              }
+            }
+            input {
+              selection {
+                direct_reference {
+                  struct_field {
+                    field: 1
+                  }
+                }
+                root_reference {
+                }
+              }
+            }
+            failure_behavior: FAILURE_BEHAVIOR_THROW_EXCEPTION
+          }
+        }
+        expressions {
+          cast {
+            type {
+              i64 {
+                nullability: NULLABILITY_NULLABLE
+              }
+            }
+            input {
+              selection {
+                direct_reference {
+                  struct_field {
+                    field: 2
+                  }
+                }
+                root_reference {
+                }
+              }
+            }
+            failure_behavior: FAILURE_BEHAVIOR_THROW_EXCEPTION
+          }
+        }
+      }
+    }
+    names: "tconst"
+    names: "avg_rating"
+    names: "num_votes"
+  }
+}
+```
+
+# Getting Started
+```
+git clone --recursive https://github.com/substrait-io/substrait-python.git
+cd substrait-python
+```
+
+# Setting up your environment
+## Conda env
+Create a conda environment with developer dependencies.
+```
+conda env create -f environment.yml
+conda activate pysubstrait
+```
+
+# Build
+## Python package
+### Editable installation
+```
+pip install -e .
+```
+
+## Generate protocol buffers
+Generate the protobuf files manually. Requires protobuf `v3.20.1`.
+```
+./gen_proto.sh
```

Comment:
Just curious: 

Could we make this part of the setup which would automatically run?

Reply:
I don't think we want to do that -- one of the conveniences here should be that the files are already generated (using the correct version of `protobuf`) and bundled into the package for the end-user.